modules/ossm-tutorial-bookinfo-overview.adoc

```diff
@@ -6,9 +6,9 @@ This CONCEPT module included in the following assemblies:
 [id="ossm-tutorial-bookinfo-overview_{context}"]
 = Bookinfo application
 
-The upstream Istio project has an example tutorial called https://istio.io/docs/examples/bookinfo[bookinfo], which is composed of four separate microservices used to demonstrate various Istio features.  The Bookinfo application displays information about a book, similar to a single catalog entry of an online book store. Displayed on the page is a description of the book, book details (ISBN, number of pages and other information), and book reviews.
+The upstream Istio project has an example tutorial called https://istio.io/docs/examples/bookinfo[bookinfo], which is composed of four separate microservices used to demonstrate various Istio features. The Bookinfo application displays information about a book, similar to a single catalog entry of an online book store. Displayed on the page is a description of the book, book details (ISBN, number of pages and other information), and book reviews.
```

Comment:
"ISBN, number of pages, and other information" - comma after pages.

Reply:
Fixed.